tensorflow/lite/micro/kernels/squared_difference.cc

```diff
@@ -124,6 +124,56 @@ TfLiteStatus SquaredDifferencePrepare(TfLiteContext* context,
         std::numeric_limits<int8_t>::min();
     data->arithmetic_params.quantized_activation_max =
         std::numeric_limits<int8_t>::max();
+  } else if (input1->type == kTfLiteInt16) {
+    const auto& input1_quantization_params = input1->params;
+    const auto& input2_quantization_params = input2->params;
+    const auto& output_quantization_params = output->params;
```

Comment:
let's not use auto here since the types are not apparent from the immediate context.

Reply:
Done.